src/core_plugins/metrics/public/components/vis_types/timeseries/vis.js

```diff
@@ -71,6 +71,11 @@ class TimeseriesVisualization extends Component {
 
     if (model.axis_min) mainAxis.min = model.axis_min;
     if (model.axis_max) mainAxis.max = model.axis_max;
+    if (model.axis_scale === 'log') {
+      mainAxis.mode = 'log';
+      mainAxis.transform = value => value >= 0 ? Math.log(value) / Math.LN10 : null;
```

Comment:
this might produce -Infinity (`Math.log(0)`), is that ok ?

Reply:
Good point... I will change `>=` to `>`